FabLabKasse/UI/ClientDialogCode.py

```diff
@@ -173,3 +185,81 @@ def reject(self):
         msgBox.setText(u"Bezahlung abgebrochen.")
         msgBox.exec_()
         QtGui.QDialog.reject(self)
+    
+    def ask_admin_pin(self):
+        cfg = scriptHelper.getConfig()
+        try:
+            pin = cfg.get('payup_methods', 'client_admin_pin');
+        except ConfigParserError:
+            msgBox = QtGui.QMessageBox(self)
+            msgBox.setText(u"fehlende Konfiguration")
+            msgBox.exec_()
+            return False
+        admin_pin = KeyboardDialog.askText("Admin PIN?", parent=self, hidden_input=True)
+        if admin_pin != "1234":
+            msgBox = QtGui.QMessageBox(self)
+            msgBox.setText(u"Falsche PIN.")
+            msgBox.exec_()
+            return False
+        return True
+        
+    def register(self):
+        if not self.ask_admin_pin():
+            return
+        username = KeyboardDialog.askText("Kundenkennung (nur a-z 0-9 -) (mind. 5 Zeichen): vorname-nachname oder firma", parent=self) or ""
```

Comment:
```suggestion
        username = KeyboardDialog.askText(u"Kundenkennung (nur a-z 0-9 -) (mind. 5 Zeichen): vorname-nachname oder firma", parent=self) or ""
```
again here; unicode?